components/Event/EventNavBar.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="RightBar-container">
+    <p>Compro Calender</p>
+    <div>
+      <EventField
+        v-for="item in props.events"
+        :key="item.id"
+        :name="item.name"
+        :image="item.image"
+        :id="item.id"
+        @click="handleClick"
+      />
+    </div>
+  </div>
+</template>
+  
+<script setup>
+const emit = defineEmits(["include", "exclude"]);
+const props = defineProps({
+  events: {
+    type: Array,
+    required: true,
+  },
+});
+
+const handleClick = (prop) => {
```

Comment:
rename prop to `eventMetaData` as we are sending customize event data from `EventFiled.vue`